atomic_reactor/plugins/pre_hide_files.py

```diff
@@ -0,0 +1,122 @@
+"""
+Copyright (c) 2019 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from __future__ import absolute_import
+
+import os
+
+from atomic_reactor.plugin import PreBuildPlugin
+from atomic_reactor.plugins.pre_reactor_config import get_hide_files
+from atomic_reactor.util import df_parser
+from atomic_reactor.constants import INSPECT_CONFIG
+
+
+class HideFilesPlugin(PreBuildPlugin):
+    key = 'hide_files'
+    is_allowed_to_fail = False
+
+    def __init__(self, tasker, workflow):
+        """
+        Plugin initializer
+
+        :param tasker: DockerTasker instance
+        :param workflow: DockerBuildWorkflow instance
+        """
+        super(HideFilesPlugin, self).__init__(tasker, workflow)
+        self.start_lines = []
+        self.end_lines = []
+
+    def run(self):
+        """
+        run the plugin
+        """
+        try:
+            self.hide_files = get_hide_files(self.workflow)
+        except KeyError:
+            self.log.info("Skipping hide files: no files to hide")
+            return
+
+        if not self.hide_files['tmpdir'] or not self.hide_files['files']:
```

Comment:
Can this be enforced in the config schema?

Reply:
ok